core/drivers/crypto/caam/caam_rng.c

```diff
@@ -556,6 +567,27 @@ enum caam_status caam_rng_init(vaddr_t ctrl_addr)
 }
 
 #ifdef CFG_NXP_CAAM_RNG_DRV
+#ifdef CFG_WITH_SOFTWARE_PRNG
+void plat_rng_init(void)
+{
+	TEE_Result res = TEE_SUCCESS;
```

Comment:
We usually initialize `res` with an error like `TEE_ERROR_GENERIC`

Reply:
I usually use `TEE_SUCCESS`. The point is to get a predictable value.

Reply:
Ok

Reply:
> My preference would be _not_ to initialize variables with unused values, so any modern compiler may warn if future code changes start using the value uninitialized.

That would be our choice if compilers weren't so poor at diagnosing "used but not initialized" variables. We have observed cases where such variables were not reported (by GCC at least).